v2/pkg/ipam/pool.go

```diff
@@ -273,3 +284,15 @@ func (p *pool) AllocateBlock(ctx context.Context, nodeName string) (*coilv2.Addr
 	p.log.Error(ErrNoBlock, "no available blocks")
 	return nil, ErrNoBlock
 }
+
+// IsUsed returns true if the pool is used by some AddressBlock.
+func (p *pool) IsUsed(ctx context.Context) (bool, error) {
```

Comment:
`pool.IsUsed` seems not to need to be exported, since it's only called from inside ipam package.

Reply:
@ysksuzuki 
As you wrote, it does not need to be exported.  I followed the names of other methods in `pool`.
I'm not sure why `pool`'s methods are capitalized.  One possibility is that they were exported via an interface originally.  Another possibility is that there was some special policy; the methods called only from within the type should be lowercased; the methods called from outside the type should be capitalized.

Reply:
So, are you going to leave it then?

Reply:
@ysksuzuki 
Yes.